kotsadm/pkg/handlers/preflight.go

```diff
@@ -279,3 +280,37 @@ func (h *Handler) PostPreflightStatus(w http.ResponseWriter, r *http.Request) {
 
 	w.WriteHeader(204)
 }
+
+func (h *Handler) SkipPreflights(w http.ResponseWriter, r *http.Request) {
```

Comment:
Function is named `SkipPreflights` but I don't see it skipping preflights.  It only sends a report.